code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -187,122 +187,3 @@
-			to_chat(M.current, "<span class='danger'>You find yourself unable to mouth the words of the forgotten...</span>")
-			M.current.remove_language(LANGUAGE_CULT)
-			M.memory = ""
-
-		if(M in ticker.mode.wizards)
-			ticker.mode.wizards -= M
-			M.special_role = null
-			M.current.spellremove(M.current, config.feature_object_spell_system? "object":"verb")
-			to_chat(M.current, "<span class='danger'><FONT size = 3>Your powers ebb and you feel weak. You are no longer a wizard.</FONT></span>")
-			ticker.mode.update_wizard_icons_removed(M)
-
-		if(M in ticker.mode.apprentices)
-			ticker.mode.apprentices -= M
-			M.special_role = null
-			M.current.spellremove(M.current, config.feature_object_spell_system? "object":"verb")
-			to_chat(M.current, "<span class='danger'><FONT size = 3>What little magic you have leaves you. You are no longer a wizard's apprentice.</FONT></span>")
-			ticker.mode.update_wizard_icons_removed(M)
-
-		if(M in ticker.mode.changelings)
-			ticker.mode.changelings -= M
-			M.special_role = null
-			M.current.remove_changeling_powers()
-			M.current.verbs -= /datum/changeling/proc/EvolutionMenu
-			if(M.changeling)
-				qdel(M.changeling)
-				M.changeling = null
-			to_chat(M.current, "<span class='danger'><FONT size = 3>You grow weak and lose your powers. You are no longer a changeling and are stuck in your current form.</FONT></span>")
-
-		if(M in ticker.mode.vampires)
-			ticker.mode.vampires -= M
-			M.special_role = null
-			M.current.remove_vampire_powers()
-			if(M.vampire)
-				qdel(M.vampire)
-				M.vampire = null
-			to_chat(M.current, "<span class='danger'><FONT size = 3>You grow weak and lose your powers. You are no longer a vampire and are stuck in your current form.</FONT></span>")
-
-		if(M in ticker.mode.syndicates)
-			ticker.mode.syndicates -= M
-			ticker.mode.update_synd_icons_removed(M)
-			M.special_role = null
-			//for (var/datum/objective/nuclear/O in objectives)
-			//	objectives-=O
-			to_chat(M.current, "<span class='danger'><FONT size = 3>Your masters are likely dead or dying. You are no longer a syndicate operative.</FONT></span>")
-
-		if(M in ticker.mode.traitors)
-			ticker.mode.traitors -= M
-			M.special_role = null
-			to_chat(M.current, "<span class='danger'><FONT size = 3>Your masters are likely dead or dying.  You are no longer a traitor.</FONT></span>")
-			if(isAI(M.current))
-				var/mob/living/silicon/ai/A = M.current
-				A.set_zeroth_law("")
-				A.show_laws()
-
-		if(M in ticker.mode.malf_ai)
-			ticker.mode.malf_ai -= M
-			M.special_role = null
-			var/mob/living/silicon/ai/A = M.current
-
-			A.remove_malf_spells()
-
-			A.laws = new base_law_type
-			A.show_laws()
-			A.icon_state = "ai"
-
-			to_chat(A, "<span class='danger'><FONT size = 3>The massive blast of energy has fried the systems that were malfunctioning.  You are no longer malfunctioning.</FONT></span>")
-		CHECK_TICK
-*/
```

Comment:
something something atomic

I won't get on your case too bad but someone had to mention it

Reply:
I just deleted all references to isapprentice I found